packages/origin-247-certificate/migrations/1646865380689-AddDeploymentPropertiesIssuer.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddDeploymentPropertiesIssuer1646865380690 implements MigrationInterface {
+    name = 'AddDeploymentPropertiesIssuer1646865380690';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
why patryk's zmiany are here

Reply:
this is not Patryk zmiany, this is to store the address of deployed issuer contract